src/button-react/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Mostly probably that we won't add any files to button-react, so I propose to rename this file to `src/button-react.js`.

Reply:
thanks, next it can be replaced with folder is that needed